Lib/test/test_codecs.py

```diff
@@ -1753,6 +1754,14 @@ def test_file_closes_if_lookup_error_raised(self):
 
             file().close.assert_called()
 
+    def test_copy(self):
+        orig = codecs.lookup('utf-8')
+        dup = copy.copy(orig)
+
+        self.assertEqual(dup.encode, orig.encode)
```

Comment:
Add `self.assertIsNot(dup, orig)`. All these tests are passed if `copy()` simply returns the original object.